cli-setup.sh

```diff
@@ -235,12 +253,17 @@ create_cli_config(){
     max_auth_retry=3
     while (( ${auth_retry} < ${max_auth_retry} )); do
 	(( auth_retry++ ))
-	echo ""
-	stdout_log "Please provide your Platform9 Credentials"
 	if [ $auth_retry -gt 1 ]; then 
 	    stdout_log "Attempt $auth_retry of ${max_auth_retry}:"; fi
-	eval "${cli_exec}" config create
-	
+
+    if [ ! -z "${PF9_MGMTURL}" ] && [ ! -z "${PF9_USER}" ] && [ ! -z "${PF9_PASS}" ] && [ ! -z "${PF9_REGION}" ] && [ ! -z "${PF9_TENANT}" ]; then
+        eval "${cli_exec}" config create --du_url ${PF9_MGMTURL} --os_username ${PF9_USER} --os_password ${PF9_PASS} --os_region ${PF9_REGION} --os_tenant ${PF9_TENANT}
+    else
+        echo ""
+        stdout_log "Please provide your Platform9 Credentials"
+        eval "${cli_exec}" config create
```

Comment:
Is the tenant name and region mandatory now? If so, this is a change in previous behavior. @chrisj-platform9 needs to update the documentation around it.

Reply:
Interactive experience for the `config create` command has the functionality to default `project` and `region` for FT users but it isn't there as part of `cli-setup.sh` after Tom's changes.

After discussion with @chrisj-platform9  , the description below is added as part of Interactive experience for the `config create` command:
```Enter Region & Tenant Details (Freedom Plan requires RegionOne and service)```

Documentation in general needs update for the modified UX.